tests/test-rest-attachments-controller.php

```diff
@@ -20,6 +20,17 @@ public function setUp() {
 		$this->contributor_id = $this->factory->user->create( array(
 			'role' => 'contributor',
 		) );
+		//matches core capabilities for subscriber role and adds upload_files
+		_x( 'File upload role', 'User role' );
```

Comment:
Right, but you are not assigning it to anything, no?

On Thu, Aug 18, 2016 at 19:00 Doug Cone

<
mailto:Doug Cone <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

In
https://github.com/WP-API/WP-API/pull/2571#discussion_r75346813
:

> @@ -20,6 +20,17 @@ public function setUp() { > $this->contributor_id = $this->factory->user->create( array( > 'role' => 'contributor', > ) ); > + //matches core capabilities for subscriber role and adds upload_files > + _x( 'File upload role', 'User role' );

well 'File upload role' in line 25 should be translated but doesn't need to be for tests I guess

—

You are receiving this because you were mentioned.

Reply to this email directly,
https://github.com/WP-API/WP-API/pull/2571/files/d3c362b55cc84892cbceeec3f0010b0b188ad218#r75346813
, or
https://github.com/notifications/unsubscribe-auth/AAJ3k9WBG4KSPOArX37QkhTV5fSeZ8vhks5qhI--gaJpZM4JBswu
.